### ISSUE TITLE
Some convert error: invalid checksum and missing 0xff00 sequence.

### ISSUE THREAD

**Issue by JerrySukin** [webp-sh/webp_server_go#137]
**Environment:**
0.4.5 with remote backend.

**Describe the bug**
These two images cause errors and do not return content when using the remote backend.

`webp-server[1949]: time="2022-08-02 03:33:51" level=error msg="image file /www/invalid_checksum.png is corrupted: png: invalid format: invalid checksum" func="[86:main.readRawImage()]"`

`webp-server[1949]: time="2022-08-02 03:36:52" level=error msg="image file /www/missing_0xff00.jpg is corrupted: invalid JPEG format: missing 0xff00 sequence" func="[86:main.readRawImage()]"`

invalid_checksum ( https://user-images.githubusercontent.com/45689550/182232953-58f76dde-9fe6-46a1-b669-f4087527c3ff.png )
missing_0xff00 ( https://user-images.githubusercontent.com/45689550/182232958-d85044b1-a9b9-426e-a8af-953e99996c6a.jpg )

**Comment by n0vad3v** [comment]
Emmm, this seems to be the problem with Golang's library, some related issues here:
* https://github.com/golang/go/issues/43382
* https://github.com/golang/go/issues/28717
* https://github.com/golang/go/issues/40130

Currently we have no idea how to fix this.

**Comment by n0vad3v** [comment]
Hi

I've made some optimization on this case as it seems there is no ETA on official fix here.

Instead of returning
```
sendfile: file /opt/exhaust/182232953-58f76dde-9fe6-46a1-b669-f4087527c3ff.png-etag-"bd736e54eb0880048e64deb49190767b" not found

**Comment by n0vad3v** [comment]
Hi

I've made some optimization on this case as it seems there is no ETA on official fix here.

After this optimization is made,when original file is corrupted in remote mode, as in this case, instead of returning
```
sendfile: file /opt/exhaust/182232953-58f76dde-9fe6-46a1-b669-f4087527c3ff.png-etag-"bd736e54eb0880048e64deb49190767b" not found
```
the original file will be returned as it is, this feature will be shipped starting with version 0.4.7.
